src/components/MyAccount/Settings/AccountSettingsDisplayOptions.tsx

```diff
@@ -3,24 +3,25 @@ import {
   FormField,
   Select,
   TextInput,
+  Box,
 } from "@nypl/design-system-react-components"
-import { notificationPreferenceMap } from "../../../utils/myAccountData"
+import { notificationPreferenceTuples } from "../../../utils/myAccountUtils"
 import type { Patron } from "../../../types/myAccountTypes"
-import { accountSettings } from "./AccountSettingsUtils"
+import { accountSettings, getLibraryByCode } from "./AccountSettingsUtils"
 import { buildListElementsWithIcons } from "../IconListElement"
 import type { JSX, ReactNode } from "react"
+import { filteredPickupLocations } from "../../../../__test__/fixtures/processedMyAccountData"
```

Comment:
Move `filteredPickupLocations` to another file. Seeing that it is located in a test file can be confusing and brings up the question of whether it is real data or not.

Reply:
good call. updated.